### ISSUE TITLE
[question] apply different configs to different files.

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint#8356]
Sometimes we need apply different configs to different files. eg. in`*.vue`, `*.jsx`,  we need some rules to check, while in `*.md`, we don't need, since it is just an example.

```js
System.import('./app.js');
```
if in `*.md`, eslint may report some error(s) like `System is not defined.` But I expect not.

As far as I know, if all the markdown files are in one directory, I can create a `.eslintrc` to override. But if not,what should I do?

**Comment by vitorbal** [comment]
@Aladdin-ADD one option would be to run ESLint multiple times, with different configs for each filetype.

In the future, you will be able to configure ESLint to override configs based on glob patterns, but unfortunately that feature is not ready yet (https://github.com/eslint/eslint/issues/8298)

**Comment by Aladdin-ADD** [comment]
thanks for your kindly answer. will it be in release ESLint@4.0?

**Comment by vitorbal** [comment]
Closing, as the issue seems to be resolved. Please reopen if there is more to discuss. Thanks!
